android/src/main/java/com/doublesymmetry/trackplayer/Utils.kt

```diff
@@ -16,11 +16,10 @@ import com.google.android.exoplayer2.upstream.RawResourceDataSource
 /**
  * @author Guichaguri
```

Comment:
there's still a few scattered around like this that we could completely remove. Unless they're still original files and you'd like to keep them.

Reply:
Yes we still use the `Utils`